csharp/ql/lib/change-notes/2024-12-12-add-markupstring-as-html-injection-sink.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added the constructor of `Microsoft.AspNetCore.Components.MarkupString` as an `html-injection` sink. This will help catch cross-site scripting resulting from using `MarkupString`. 
```

Comment:
Nit: it's not just the constructor that's added as a sink.

Reply:
Modified the wording to mention the "explicit cast operator". Is that appropriate wording for `op_Explicit`?